concepts/identity.md

```diff
@@ -0,0 +1,284 @@
+# META Identity
+
+This document outlines the concept of a META Identity. A META Identity is used
+to both identify participants in the network using a cryptographic identifier
+and to provide verifiable information about those participants.
+
+## Status of this Document
+
+The META protocol is at an early stage of development, expect this document to
+change frequently during this early phase.
+
+## Outline
+
+A META Identity is a representation of a participant in the META network. It is
+designed to be a self-sovereign identity with two functions:
+
+- Referable unique identifier for a participant within the META network
+
+- Enable web of trust mechanisms through inclusion and signing of external
+  identifiers relating to the participant (eg. META-ID = [Spotify-ID, IPI,
+  National ID, PGP key, Twitter handle])
+
+## Terminology
+
+- Custodian - an Ethereum address appointed by the META Identity owner and granted
+  certain privileges in order to act on the owner's behalf.
+- Identity Custody Scheme - system for controlling the ownership of META Identities.
+- Identity Owner - an Ethereum address registered as the owner of a META Identity.
+- Identity Signature Scheme - system for controlling the cryptographic signing of
+  META Identity metadata.
+- Smart Contract - a collection of code (functions) and data (state) that resides
+  at a specific address on the Ethereum blockchain.
+- Trust Item - a detached cryptographic signature of a metadata item.
```

Comment:
Thanks for adding these descriptions, @lukehedger . Actually I interpret Trust Item as the actual link between those two entities

Reply:
Full disclosure, I stole that description from further down the document, under the heading Services>Signing
> A Trust Item in META Identity SHOULD be a detached cryptographic signature of a metadata item.

Could change to "a link between a cryptographic signature and a metadata item."?

Reply:
Should work.